t3f/initializers.py

```diff
@@ -7,6 +7,53 @@
 from t3f import shapes
 
 
+def _validate_input_parameters(**params):
```

Comment:
Make is_tensor and shape required params:
_validate_input_parameters(is_tensor, shape, **params):